.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: 'enhancement'
+assignees: '@agraebe'
```

Comment:
Could you assign me instead?

Reply:
Better actually would be to keep this blank and create a CODEOWNERS in this repo to auto-assign someone.

Reply:
Removed this line.